src/app/MoreGuide/page.tsx

```
@@ -0,0 +1,244 @@
+  gap: 20px;
+`;
+
+const Title = styled.h1`
+  margin: 0;
+  color: ${color.black};
+  ${font.D1};
+`;
+
+const Date = styled.span`
+  color: ${color.gray500};
+  font-family: ${font.p1};
```

Biome (2.1.2)

[error] Do not shadow the global "Date" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)